db/schema.rb

```diff
@@ -240,6 +240,7 @@
     t.string   "static_emails_on_automated_deploy_failure",    limit: 255
     t.string   "datadog_monitor_ids",                          limit: 255
     t.string   "jenkins_job_names",                            limit: 255
+    t.integer  "next_stage_id"
```

Comment:
How do you feel about reversing the direction of this association from a `belongs_to` to a `has_many`? That could allow for a tree of host groups, e.g. after pod 1, deploy to pods 2, 3, and 5 in parallel.